js/utils/musicutils.js

```diff
@@ -1992,6 +1992,10 @@ function _getStepSize(
 function _buildScale(keySignature) {
     var obj = keySignatureToMode(keySignature);
     var myKeySignature = obj[0];
+    if(myKeySignature == "C" + FLAT) {
```

Comment:
We have a similar issue with B#, which should map to C.

Reply:
Not sure if we need to support B# Major (scale). As for the B# as a pitch, yes, there are various scales that use B# that we should support.

Reply:
When I look at the code again, it looks like at makes a key/mode in B# equivalent to a key/mode in C. Is that correct? 

@aviral243 I think you understand this better than anyone. Is this something that would benefit us?

FWIW, I tested B# major in the following implementation on rawgit and it seems to do what I expect.

![Screenshot at 2020-08-27 09:27:16](https://user-images.githubusercontent.com/13454579/91448178-8fbbe800-e847-11ea-9d6e-a6de380670e0.png)

Reply:
It is correct. So B# Major is a scale that is not on the circle of fifths, so it is not a standard scale. Following the Whole whole half whole pattern of major scales, B# would have the notes: B#, Cx, Dx, E#, Fx, Gx, Ax, and B#. These are the same notes as a scale in C major. B# is equivalent to C, so it makes sense that their scales are the same.